crates/core/README.md

```diff
@@ -0,0 +1 @@
+../../README.md
```

Comment:
i assume this is a symlink and github is just being weird

Reply:
yeah idk why they're not marked